README.md

```diff
@@ -1,2 +1,102 @@
-# terraform-module-postgres
-postgres setup on Amazon AWS using Terraform and Ansible
+### 1. Introduction
+This repository contains tooling for deploying efs in Amazon AWS using terraform and ansible.
+
+### 2. Context
+This repo is part of a parent project to achieve the ability of possibility of controlled deployments in different environments so, that we can rollback
+
+### 3. Problem Statement 
+We need infrastructure modules to be pulled by the driver pipeline to be consistent to a standard format so that it can execute them in one job and push their states to output repos.
+
+### 4. Solution 
+
+The solution is to pack the CI, CD, and destroy scripts consuming terraform module in ansible playbooks and provide the consistent interface to driver repo.
+
+#### Technology Stack
+
+* Ansible
+* Bash
+* Terraform
+
+##### Infrastructure
+
+This project can be run from any type of host with no platform restrictions. However, it was originally intended to be used by a Jenkins job.
+
+### 5. Usage 
+
+#### Running
+
+0. You need to install ansible and terraform first, or use this docker image
+```
+$ docker run -it -v ${PWD}:/tmp stakater/pipeline-tools:1.0 sh
+$ cd /tmp
+```
+
+1. Run these,
+
+```
+$ mkdir -p build
+$ touch tfvars.sh
+$ touch secrets.sh
```

Comment:
Add a one line description to demonstrate the difference in b/w secrets.sh and tfvars.sh

Reply:
done